x/inflation/keeper/hooks.go

```diff
@@ -80,6 +83,37 @@ func (k Keeper) AfterEpochEnd(ctx sdk.Context, epochIdentifier string, epochNumb
 		k.SetEpochMintProvision(ctx, newProvision)
 	}
 
+	defer func() {
+		if mintedCoin.Amount.IsInt64() {
+			telemetry.IncrCounterWithLabels(
+				[]string{types.ModuleName, "hook", "allocate", "total"},
```

Comment:
let's remove the `"hook"`. I don't think it's necessary as the user / observer only cares about the allocation